### PR TITLE
document old way of checking validity of CSRF token

### PR DESCRIPTION
**Pull request by snoek09**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | yes |
| Applies to | all |
| Fixed tickets | Related to #4668 |

### CODE REVIEW

File: book/controller.rst [hunk 1]
```diff
@@ -786,6 +786,19 @@ Just like when creating a controller for a route, the order of the arguments of
 order of the arguments, Symfony will still pass the correct value to each
 variable.
 
+Checking the Validity of a CSRF Token
+-------------------------------------
+
+Sometimes you want to use CSRF protection in an action where you don't want to use a
+Symfony form.
+
+If, for example, you're doing a DELETE action, you can use the :method:`Symfony\\Component\\Form\\Extension\\Csrf\\CsrfProvider\\CsrfProviderAdapter::isTokenValid`
```
Comment (xabbuh) [inline book/controller.rst]: Can you please move the `method` role on its own line?

**Comment by snoek09** [comment]
@xabbuh done, please let me know if this needs any other changes.

File: book/controller.rst [hunk 2]
```diff
@@ -786,6 +786,20 @@ Just like when creating a controller for a route, the order of the arguments of
...
 
+Checking the Validity of a CSRF Token
+-------------------------------------
+
+Sometimes you want to use CSRF protection in an action where you don't want to use a
+Symfony form.
+
+If, for example, you're doing a DELETE action, you can use the
+:method:`Symfony\\Component\\Form\\Extension\\Csrf\\CsrfProvider\\CsrfProviderAdapter::isTokenValid`
+method to check the CSRF token::
```
Comment (xabbuh) [inline book/controller.rst]: I feel that this sentence should be worded more "smoothy". Though I'm not sure how. Maybe our native speaker @weaverryan has a suggestion.

**Comment by xabbuh** [comment]
@snoek09 I am sorry for the delay here. Do you think we should still tackle this and reopen the PR in that case?

**Comment by snoek09** [comment]
I closed this because it is already merged into 2.6: https://github.com/symfony/symfony-docs/commit/85dc2943e0ffcfbb43dbde00129d514ec0fc9000

**Comment by xabbuh** [comment]
Indeed, but it seems to be missing in the 2.7 version of the docs if I don't miss anything.

**Comment by xabbuh** [comment]
see #5816

**Comment by snoek09** [comment]
In that case, just re-use the text and change it where needed.

**Comment by xabbuh** [comment]
@snoek09 Sorry for poking you again. I had a look at this again and I think we should actually merge this one so that we have a description in the 2.3 docs too. What do you think?

**Comment by snoek09** [comment]
You are right, that's why this was added. I remember now :smile: 

Anyhow, as I deleted my fork a while back and closed this PR I can't seem to reopen this.
I can create a new PR if you want.

**Pull request closed without merging**